src/types/destinationTransformation.ts

```diff
@@ -235,6 +236,32 @@ export type DeliveryV1Response = {
   response: DeliveryJobState[];
 };
 
+/**
+ * Interface for response parameters in network handlers
+ */
+export interface ResponseParams {
+  destinationResponse: {
+    status: number;
+    response?: any;
+    headers?: any;
+  };
+  destinationRequest?: ProxyRequest;
+  destType?: string;
+  rudderJobMetadata: ProxyMetdata | ProxyMetdata[];
+  [key: string]: any;
+}
+
+/**
+ * Interface for response object from network handlers
+ */
+export interface ResponseObject extends Partial<DeliveryV0Response>, Partial<DeliveryV1Response> {
```

Comment:
ResponseObject is too generic, it doesn't convey for which API/use-case this type belongs to. following works ?
```suggestion
export interface ResponseProxyObject extends Partial<DeliveryV0Response>, Partial<DeliveryV1Response> {
```